parm/soca/obs/config/icefb_GDR.yaml

```diff
@@ -0,0 +1,39 @@
+obs space:
```

Comment:
As pointed out by @ShastriPaturi , that file needs to be renamed `icefb_GDR.yaml` ... Not your bug @AndrewEichmann-NOAA , but we might as well take cor of it now.

Reply:
You mean renamed from `icefb_GDR.yaml` to `icefb_gdr.yaml`, replacing the existing` icefb_gdr.yaml`?

Reply:
done